mlir/include/mlir/Dialect/OpenMP/OpenMPOpsInterfaces.td

```diff
@@ -142,6 +142,40 @@ def LoopWrapperInterface : OpInterface<"LoopWrapperInterface"> {
   ];
 }
 
+def ComposableLoopWrapperInterface : OpInterface<"ComposableLoopWrapperInterface"> {
+  let description = [{
+    OpenMP operations that can wrap a single loop nest. When taking a wrapper
+    role, these operations must only contain a single region with a single block
+    in which there's a single operation and a terminator. That nested operation
+    must be another loop wrapper or an `omp.loop_nest`.
```

Comment:
This description needs updating. It should say that this is for OpenMP operations that can represent a single leaf of a composite OpenMP construct. It would also be good to use that description to update this PR's description, so it's easier for reviewers to understand what this change is for.